### DISCUSSION TITLE
Not Able to Reset my password

### DISCUSSION THREAD

**Discussion by arpitpaltani21** [community/community#57314] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Not sure if it's me but i am not able to reset my password because captcha is not loading.

**Comment by DSDmark** [comment]
Hi @arpitpaltani21,

There are a few steps you can try to resolve the issue:

- Clear your browser cache and cookies.

- Try a different browser or device.

- Disable browser extensions.

- Check your internet connection.

**Reply by serep804** [reply]
my brother try all that, and it does not working because api.captcha is not load (status: 526)

**Comment by ghost** [comment]
I am also facing exact same issue while I try to reset my GitHub password and also while I try to create a ticket to GitHub support. Basically, where ever captcha is required on GitHub.com, all those features are blocked.

I think it has to be fixed by GitHub.

<img width="960" alt="image" src="https://github.com/community/community/assets/108675325/080be0b6-e8f8-4995-a495-00f7f83cb94a">

Reported by multiple people in different discussions.
https://github.com/orgs/community/discussions/57322
https://github.com/orgs/community/discussions/57317

**Reply by salomj** [reply]
I am having this issue on firefox specifically. When I try on chrome, it just says Opps, something went wrong.

**Reply by salomj** [reply]
Didn't work on Chrome nor Firefox. But it did work on Microsoft Edge.

**Comment by kanishk779** [comment]
I forgot the password for my account and tried resetting the password by using forgot password. But the page through which the reset password mail is sent, is not able to load captcha.

I have also tried the troubleshooting [guide](https://docs.github.com/en/get-started/using-github/troubleshooting-connectivity-problems#troubleshooting-the-captcha), but everything mentioned there seems to be working fine. I have also cross-checked this issue on other machines.

Could you please resolve this issue with Github.

**Reply by ghost** [reply]
Captcha has now started working on GitHub.com.

**Comment by ghost** [comment, marked as answer]
Captcha has now started working on GitHub.com.

**Reply by FransiskusAgapa** [reply]
It is still down, I've been trying to reset my password for almost a month now. Please github developers, do something dudes...🥲

**Reply by salomj** [reply]
Negative on this one.

**Comment by skilling62** [comment]
The Captcha that they have put on the password reset page is a joke... have fun doing this 15 times only for it to reject it at the end. The audio option is just as bad. Both should be removed
![Dice](https://github.com/community/community/assets/22370853/b2bc9284-a1b5-40ef-bee2-e4656c3be035)
![Result](https://github.com/community/community/assets/22370853/cf0f62aa-3fc1-4ab8-b1e3-d42ca3f8c037)

**Reply by ScareCrow279** [reply]
I agree. I was entering github desktop and solve the captcha. Tried to beat the first one with dices, but my counting speed not fast enough to complete it. Tried sound one too, but after completing the captcha it just says "Committing your answer. Please wait..." and not doing anything.

**Reply by imstewart** [reply]
This is total joke of a password reset, first you have to try and count to 14 quickly using numbers and dots, then they change the rules if you are not fast enought from 4 answers to 10 answers then you run out of time. I only managed to get in, because I had my password cached on another computer. This password reset process this discrimination of the first order.... 

If you want to do this make 5 answers with a longer time, not everyone can do this sums quickly and under pressure.

**Reply by microcrit** [reply]
I lost my username and password for my personal account, I can't even get through the captcha after 10 tries, both audio and the dice sums. I thought it was something wrong with my device or network. Unbelievable.

**Reply by nrettberg** [reply]
This has not been fixed, and I am beyond aggravated at this point to put it very mildly. Why in the world, would GitHub ask me 15 times, 15 bleeping times, not just two or three times, no, it has to be 15 times(!), in a row to sum up, up to 10 different dice captchas per try. I do not steal my time and this is totally absurd! And if I get one wrong, I have to start over and do another 15 tries? My level of bleeped-off-ness should be enough to prove I am human. Do you want people to use your platform or not? I get the need to prove that someone is human to prevent the creation of some fake accounts, but my deity, this is just ridiculous. You need to realize that, yes, at some point machines will be better than humans. What are you going to do then, make me solve Einsteins equations to merge with Quantum physics just to set up an account. I am applying for a GitHub account, not for a top secret security clearance. Thanks, but I pass! But maybe that is what you want?

**Comment by prachi-baisoya09** [comment]
![image](https://github.com/community/community/assets/117898211/f7d494bd-c454-4b8d-8334-aafa73718ba4)
have successfully solved the puzzle many times still, not able to reset my password, getting this error each time, please help.

**Reply by prachi-baisoya09** [reply]
any response on it ?

**Reply by FraserElectronics** [reply]
I am having the same issue - what an absolute joke the Captcha is now.
Classic Microsoft, as soon as they take over it becomes a complete shambles.
Shame on MS Github devs for not actually helping people with this problem.

**Reply by salomj** [reply]
So, it didn't work for me on Chrome. I got a different error. It didn't work for me on Firefox, I got that error. It did work for me on Microsoft Edge.

**Reply by FraserElectronics** [reply]
I have only been using Edge 😒

**Reply by salomj** [reply]
I pretty much never use edge, and often use firefox and chrome. So maybe it is just a browser without much data stored in it? Try a browser you've never used before, such as Opera, Chromium (Not Chrome), Brave, Tor?

And be sure to not import settings from another browser.

**Reply by AndrewFraser-Tendeka** [reply]
@salomj Great shout - switched to Firefox and it worked just fine 😊
Oh the irony that it does not work with Microsoft's own browser 🤦‍♂️

Thanks for the pointer.

Andy

**Comment by salomj** [comment]
Worked for me when using Microsoft Edge.

Chrome said, "Opps something went wrong."
Chrome in private mode, said the same thing.
Firefox said, "Unable to verify captcha response." And sent me to that support documentation. 
Firefox in private mode said the same thing.
Microsoft Edge said, "Looks a-okay to me."
So, I guess, just keep trying different browsers until it works.

Some Common Browsers  (Hopefully one will work for you):
Chrome, Firefox, Edge, Safari, Opera, Chromium, Brave, Tor

**Reply by siluphye** [reply]
It works on Firefox.
